app-modules/caseload-management/database/factories/CaseloadFactory.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Assist\CaseloadManagement\Database\Factories;
+
+use App\Models\User;
+use Assist\CaseloadManagement\Models\Caseload;
+use Assist\CaseloadManagement\Enums\CaseloadType;
+use Assist\CaseloadManagement\Enums\CaseloadModel;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends Factory<Caseload>
+ */
+class CaseloadFactory extends Factory
+{
+    /**
+     * @return array<string, mixed>
+     */
+    public function definition(): array
+    {
+        return [
+            'name' => fake()->words(asText: true),
+            'model' => fake()->randomElement(CaseloadModel::cases()),
+            'type' => CaseloadType::Dynamic, //TODO: add static later
```

Comment:
Is static still later?

Reply:
I've technically added it, but I've left it disabled until we decide exactly how it's supposed to work.